app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split()
+    if cp != "cp":
+        return
+    if file == file_copy:
+        return
+    with open(file, "r") as file_in, open(file, "w") as file_out:
```

Comment:
Другий файл не відкривається тут двічі file використовується
```suggestion
    with open(file, "r") as file_in, open(file_copy, "w") as file_out:
```